code/datums/martial/cqc.dm

```diff
@@ -225,10 +225,19 @@
 ///Subtype of CQC. Only used for the chef.
 /datum/martial_art/cqc/under_siege
 	name = "Close Quarters Cooking"
-	var/list/valid_areas = list(/area/service/kitchen, /area/service/cafeteria, /area/service/bar, /area/service/theater)
+	var/list/kitchen_areas
 
-///Prevents use if the cook is not in the kitchen.
-/datum/martial_art/cqc/under_siege/can_use(mob/living/owner) //this is used to make chef CQC only work in kitchen
-	if(!is_type_in_list(get_area(owner), valid_areas))
+/// Refreshes the valid areas from the cook job singleton, otherwise uses the default kitchen area as a fallback option. See also [/datum/job/cook/New].
+/datum/martial_art/cqc/under_siege/proc/refresh_valid_areas()
+	var/datum/job/cook/cook_job = SSjob.GetJobType(/datum/job/cook)
+	if(!istype(cook_job))
+		kitchen_areas = list(/area/service/kitchen)
```

Comment:
Why are you reverting my PR entirely?

I checked the defines, the bar is only in one of the maps. I should be able to CQC in the bar/kitchen area as chef.

Reply:
ah lmao nvm this is a classic pathing fuckup, see op about the bar/atrium thing